src/main/java/frc/robot/subsystems/intake/IntakeLoggedInputs.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.subsystems.intake;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class IntakeLoggedInputs implements LoggableInputs {
+    static double supMAX;
+    static double infMAX;
+    static boolean solenoid;
```

Comment:
These variables should not be static. They should be dynamic variables, and once the 'IntakeLoggedInputs' class is instantiated you can update the variables in the `updateInputs()` method by accessing them through the instance.

Reply:
done.